src/software/pipeline/main_cameraInit.cpp

```diff
@@ -471,6 +507,32 @@ int aliceVision_main(int argc, char **argv)
     const double diag24x36 = std::sqrt(36.0 * 36.0 + 24.0 * 24.0);
     camera::EIntrinsicInitMode intrinsicInitMode = camera::EIntrinsicInitMode::UNKNOWN;
 
+
+    std::unique_ptr<oiio::ImageInput> in(oiio::ImageInput::open(view.getImagePath()));
+
+    std::string imgFormat = in->format_name();
+
+    // check if a color profile is available for the image
+    if (!colorProfileDatabaseDirPath.empty() && (imgFormat.compare("raw") == 0))
+    {
+        bool colorProfileFound = false;
+        if (hasCameraMetadata)
+        {
+            std::pair<std::string, std::string> pairMakeModel = std::pair<std::string, std::string>(make, model);
```

Comment:
I think it makes sense to create a small struct for make and model, because taking members as `pair.first` results in very non-descriptive code.